src/javascript/lib/landing_pages/ico.js

```diff
@@ -431,3 +436,10 @@ function setupCrowdin() {
         }
     }
 }
+
+function getDocumentUrl(lang = 'en') {
+    if (lang === 'en') {
+        return 'https://ico_documents.binary.com/howto_ico.pdf';
+    }
+    return `https://ico_documents.binary.com/howto_ico_${lang}.pdf`;
```

Comment:
this is wrong - only ru has its separate PDF. Other languages should default to English.